kinde_sdk/auth/route_protection.py

```
@@ -0,0 +1,434 @@
+            
+            # Load global settings
+            self.global_settings = config.get("settings", {})
+            
+            # Load route rules
+            routes_config = config.get("routes", {})
+            self.routes = {}
+            
+            for route_name, route_config in routes_config.items():
+                # Validate required fields
+                if "path" not in route_config:
+                    raise ValueError(f"Route '{route_name}' missing required 'path' field")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)